RUFAS/simulation_engine.py

```diff
@@ -176,3 +183,9 @@ def _initialize_simulation(self) -> None:
         self.manure_manager = ManureManager(all_pen_manure_data, self.weather, self.time, manure_class_config)
 
         self.field_manager = FieldManager(manure_manager=self.manure_manager, feed_manager=self.feed_manager)
+
+        run_end_to_end_testing = self.im.get_data("end_to_end_testing_inputs")
+        if not run_end_to_end_testing:
+            return
+        self.feed_manager.setup_stored_feeds(run_end_to_end_testing, self.time)
+        self.run_end_to_end_testing = True
```

Comment:
```suggestion
        if run_end_to_end_testing is not None:
            self.feed_manager.setup_stored_feeds(run_end_to_end_testing, self.time)
            self.run_end_to_end_testing = True        
```
I think this is more readable. It is also less susceptible to bugs.